### ISSUE TITLE
Entire site is unreadable/unusable on mobile browsers

### ISSUE THREAD

**Issue by vertigan** [Raku/doc-website#411]
No biggie, it's only 60% of the market you're losing right now. Fortunately there's a very simple fix, add
<meta name="viewport" content="width=device-width, initial-scale=1.0"> in the <head> section of each page and it becomes possible to learn Raku on the toilet again. I've attached two screen shots to show the difference but don't be fooled by the relative readability of the withoutvp.jpg on your monitor it's so much worse reading on a phone.
![withoutvp](https://github.com/user-attachments/assets/0e1c0ce1-2fbb-4d92-8b02-86328e496546)
![withvp](https://github.com/user-attachments/assets/12263cfb-113e-4a80-9799-e66f750f8505)

**Comment by finanalyst** [comment]
@vertigan @coke This is a known problem, and it annoyed me too that I couldn't use a mobile to check the site. Its mostly solved by upgrading to Bulma v1, which is advertised to be mobile oriented. Currently site uses Bulma v.0.9, but the upgrade needs quite a lot of work, so the upgrade will be incorporated in a website upgrade that uses RakuAST. Good news, this upgrade is quite close.

For an example of RakuDoc sources rendered into HTML using RakuAST and Bulma v1, look at [module documentation](https://finanalyst.github.io) I've tested this on my mobile and it seems to work well.

**Comment by finanalyst** [comment]
@coke @vertigan Status update on this issue: Please take a look at [new-raku](https://new-raku.finanalyst.org) and try looking at the site on a mobile.
(There are not too many pages yet, as this remains a work in progress).
It seems to me to be a lot better now.

**Comment by vertigan** [comment]
@finanalyst yes it's looking much better, the source examples are legible and the navigation seems mostly coherent. My only criticism would be the menu has all the categories expanded with no way to collapse them and it's very unclear to me why "Switch UI" would be the place to switch language, I can't imagine how a non-native speaker would make the connection. Having the the current language, with a flag icon would better match the current paradigm of where people expect to find that.

Overall though while I'm happy to consider the issue closed I feel compelled to mention that the old site was also legible with the addition of a single header tag so ubiquitous that google analytics will send you nagging emails if it's not present. The fact that no one thought a perl, awk or even raku one-liner to do something like 
s/<head>/<head\n<meta name="viewport" content="width=device-width, initial-scale=1">/
was worth doing in the previous 9 months  while Bulma was being worked on casts serious doubts on Raku's  feasiblility for... well anything really.
